src/sentry/api/endpoints/project_ownership.py

```diff
@@ -46,6 +46,7 @@ def validate(self, attrs):
                     bad_actors.append(u"#{}".format(owner.identifier))
 
         if bad_actors:
+            bad_actors.sort()
```

Comment:
Seems like this would put teams to the top at all times now. Do we want/need this (except for tests?)

Reply:
IMO this is fine. It's just a validation message, having it be guaranteed to be stable is probably vaguely better. I don't think it matters a lot of the teams are at the top.